### DISCUSSION TITLE
Adding ChatGPT API

### DISCUSSION THREAD

**Discussion by jucasoliveira** [jucasoliveira/terminalGPT#59] [General]
Recently , openAI released a new API we can definetly use for our terminalGPT:

https://platform.openai.com/docs/guides/chat/introduction

This will be able to us:
- remove the fineTunning, on which is costly. 
- Use the JSONL file to add context to the converstation and return the result.
I'll be working on this  tomorrow, let's see whats gonna happen.

**Comment by LamontOmlanduH9** [comment]
That sounds exciting! Using OpenAI's new API could potentially save you a lot of time and resources. I wish you the best of luck with your implementation and look forward to hearing how it goes.

**Comment by jucasoliveira** [comment]
New chatGPT API added to terminalGPT
